pkg/auth/rbac.go

```diff
@@ -185,6 +191,43 @@ func ValidateRbacPermission(line string) (p Permission, err error) {
 	}, nil
 }
 
+func ValidateTeamRbacPermission(line string, teamPermissions *RBACTeams) (team string, err error) {
+
+	p := strings.Split(line, ",")
+
+	if len(p) != 2 {
+		return "", fmt.Errorf("2 fields are expected but %d were specified", len(p))
```

Comment:
please return the whole line in the error, e.g.
```suggestion
		return "", fmt.Errorf("2 fields are expected but %d were specified in line %s", len(p), line)
```